iModelCore/ECPresentation/Source/Shared/Queries/CustomFunctions.cpp

```diff
@@ -1373,6 +1373,24 @@ struct JsonArrayAggregate : ECPresentation::AggregateFunction
 };
 #endif
 
+/*=================================================================================**//**
+* Parameters:
+* - binary value
+* @bsiclass
++===============+===============+===============+===============+===============+======*/
+struct GuidToStrScalar : ECPresentation::ScalarFunction
+    {
+    GuidToStrScalar(CustomFunctionsManager const& manager)
+        : ScalarFunction(FUNCTION_NAME_GuidToStr, 1, DbValueType::TextVal, manager)

```

Comment:
@khanaffan, it seems the `GuidToStr` function, along with some others, is only registered with the primary connection, which means we can't use them on any secondary connections. Is that by design? Could they be made available on all connections?

Reply:
Resolving this conversation to avoid blocking the PR. Will talk to @khanaffan separately.